src/WebJobs.Script/Diagnostics/FileTraceWriter.cs

```diff
@@ -207,7 +205,7 @@ internal void SetNewLogFile()
 
             // we include a machine identifier in the log file name to ensure we don't have any
             // log file contention between scaled out instances
-            string filePath = Path.Combine(_logFilePath, string.Format(CultureInfo.InvariantCulture, "{0}-{1}-{2}.log", DateTime.UtcNow.ToString("yyyy-MM-ddTHH-mm-ssK"), _instanceId, _processId));
```

Comment:
The addition of the process ID is actually what caused the regression because we do a file search to find existing log files using pattern "*-{0}.log" where the parameter is the instance ID. The change to the file format broke this. We could also update the pattern, but I chose to remove process ID from the file name since I don't see how it adds anything.